src/components/Button/ButtonNavigation/index.jsx

```diff
@@ -0,0 +1,46 @@
+const ButtonNavigation = () => {
```

Comment:
Coba dijelasin kenapa buttonNavigation ada component sendiri
Component kan reusable anywhere tanpa tau dia contextnya dipake dimana, kalo button navigation ini bakal spesifik ke navigation kan
IMO mending bikin subsection baru
Jadi ada low level component yang reusable anywhere dan ada UIkit yang kek navigation buttons
Jadi UIkit ini ngepake beberapa component, instead of component dia itu lebih ke UI level yang attached ama spesifik UI
Nah ButtonNavigation ini bisa pake CommonButton
Jadi mungkin di CommonButton di modify biar bisa nerima children element ato outlet